install/deb/templates/web/nginx/php-fpm/wordpress_mu_subdir.stpl

```diff
@@ -0,0 +1,100 @@
+#=========================================================================#
+# Default Web Domain Template                                             #
+# DO NOT MODIFY THIS FILE! CHANGES WILL BE LOST WHEN REBUILDING DOMAINS   #
+# https://hestiacp.com/docs/server-administration/web-templates.html      #
+#=========================================================================#
+
+server {
+	listen      %ip%:%web_ssl_port% ssl;
+	server_name %domain_idn% %alias_idn%;
+	root        %sdocroot%;
+	index       index.php index.html index.htm;
+	access_log  /var/log/nginx/domains/%domain%.log combined;
+	access_log  /var/log/nginx/domains/%domain%.bytes bytes;
+	error_log   /var/log/nginx/domains/%domain%.error.log error;
+
+	# Rewrite requests to /wp-.* on subdirectory installs.
+	if (!-e $request_filename) {
+		rewrite /wp-admin$ $scheme://$host$uri/ permanent;
+		rewrite ^/[_0-9a-zA-Z-]+(/wp-.*) $1 last;
+		rewrite ^/[_0-9a-zA-Z-]+(/.*\.php)$ $1 last;
+	}
+
```

Comment:
@hudokkow Suggested to move before "location /" for consistency with other default templates format/style.

Also NGINX's "if" as special location block, which also avoids priority conflicts with other functions.

Reply:
See you already did the changes in the other branch. Fine by me.

Also no need to worry with attribution. Squash away... ;-)

Thanks!

Reply:
Will add a attribution in the Changeling :)